content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Shutting Down and Restart Cluster Gracefully"
+description: "Demonstrate how to shut down and restart kubernetes cluster Gracefully"
+layout: "single"
+
+linkTitle: "Shutting Down and Restart Cluster Gracefully"
+weight: 2000
+
+icon: "/images/docs/docs.svg"
+---
+This document describes the process to gracefully shut down your cluster. You might need to temporarily shut down your cluster for maintenance reasons.
+
+{{< notice warning >}}
+Shutting down cluster is very dangerous, you must to understand what are you doing and make an etcd backup before proceeding.
+Usually we recommend one by one maintenance nodes instead of restarting cluster.
+{{</ notice >}}
+
+### Prerequisites
+Take an [etcd buckup](https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/recovery.md#snapshotting-the-keyspace) prior to shutting down the cluster.
+Ssh [keyless login](https://man.openbsd.org/ssh.1#AUTHENTICATION) is set up between hosts.
+
+### Shutting Down the Cluster
+{{< notice warning >}}
+You must to take an etcd backup before proceeding for your cluster can be restored if you encounter any issues when restarting the cluster.
+{{</ notice >}}
+
+{{< notice tip >}}
+Use this method can shut down the cluster gracefully ,but there is still the possibility of data corruption.
```

Comment:
->
Using the method in this tutorial can shut down a cluster gracefully, while the possibility of data corruption still exists.